cg/cli/upload/delivery_report.py

```diff
@@ -107,9 +111,13 @@ def delivery_report(context, family_id, print_console, force_report):
 
     click.echo(click.style("----------------- DELIVERY_REPORT -------------"))
 
-    def _add_delivery_report_to_hk(delivery_report_file, hk_api: hk.HousekeeperAPI, family_id):
+    def _add_delivery_report_to_hk(
+        delivery_report_file, hk_api: hk.HousekeeperAPI, family_id, analysis_date
```

Comment:
Should we not use type hints for all arguments? It is a bit unclear what type analysis date have here. Also please specify the return type

Reply:
Oh and a dockstring.  The name of the function implies that it will add something to housekeeper but it looks like it returns stuff, makes it a bit tricky to follow what is going on